extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -107,6 +115,10 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 
 			if (window.wgEnableNewAuthModal) {
 				open(params.onAuthSuccess);
+				track({
+					action: Wikia.Tracker.ACTIONS.CLICK,
+					label: params.origin
```

Comment:
There is no params validation. Please make sure this attribute is always passed or use some default value here.